api/PaymentResponse.json

```diff
@@ -238,7 +238,7 @@
               "version_added": "12.1"
             },
             "safari_ios": {
-              "version_added": "11.3"
```

Comment:
Can you check where this data came from? Maybe it's correct? :)

Reply:
This came from https://github.com/mdn/browser-compat-data/pull/6679/files#diff-ebf770809c04c0bcb7fed9d6007fea3b3dd550547b7b6c2831e79faf8479a494L352-R237, but in the same PR, I set Safari Desktop to "12.1", so I am confident this was originally a copy-paste error.